quipucords/scanner/network/processing/process.py

```diff
@@ -92,9 +92,10 @@ def process(scan_task, previous_host_facts, fact_key, fact_value, host):
     # outer for loop.
     try:
         for dep in getattr(processor, DEPS, []):
-            if dep not in previous_host_facts or \
-                    not previous_host_facts[dep] or \
-                    isinstance(previous_host_facts[dep], Exception):
+            if not previous_host_facts or \
```

Comment:
Do we ever expect `previous_host_facts` to be false in a way that isn't an empty dictionary? If it's an empty dict, I think the next test should catch that dep is not in it. This totally could be right, I just want to understand what's going on.

Reply:
You're right. If I send `self._ansible_facts[host] from the inspect_callback.py then the check does catch it. I was only sending `self._ansible_facts` and checking `if dep not in previous_host_facts[host]` which was getting a key error which is why I added the extra check. However, I will do it the other way.